java/opendcs/src/main/java/opendcs/dai/EnumDAI.java

```diff
@@ -21,7 +21,7 @@
  * Data Access Interface for database-resident enumerations
  * @author mmaloney Mike Maloney
  */
-public interface EnumDAI extends DaiBase, OpenDcsDao
+public interface EnumDAI extends DaiBase
```

Comment:
Same comment as DaiBase; was intentionally done but I see no reason to change it in the current refactor branch as the long term affect is intended to be the same.